src/foam/u2/view/LoginView.js

```diff
@@ -220,7 +227,17 @@ foam.CLASS({
   methods: [
     function init() {
       // Use passed in values or default loginVariables defined on ApplicationControllers
-      this.param.dao_ = !! this.param.dao_ ? this.param.dao_ : this.loginVariables.dao_;
+      this.param = Object.assign(this.loginVariables, this.param)
+
+      if ( this.modelCls_ ) {
+        try {
+          this.model = (foam.lookup(this.modelCls))?.create(this.param, this);
+          return;
+        } catch (err) {
+          console.warn('Error occurred when looking up modelCls', this.modelCls, err);
+        }
+      }      
+
       // Instantiating model based on mode_
       if ( this.mode_ === this.MODE1 ) {
```

Comment:
Wouldnt this override the modelCls_?

Reply:
this should be fine since there's no default value for mode_